controllers/aws_config_loader.go

```diff
@@ -42,15 +41,24 @@ func NewAwsConfigLoaderForEnv(env venv.Env) AwsConfigLoader {
 	}
 }
 
+// CreateNewAWSSessionFromConfig returns an AWS session using AWS Config
+func CreateNewAWSSessionFromConfig(cfg aws.Config) *awssession.Session {
```

Comment:
nit: Line 57 uses `Aws`. I would personally prefer all references to AWS be `AWS` and not `Aws`. But if that's too much work, lets stick to the existing use.

Reply:
ya i agree, `AWS` is better 
changed all the functions to use that acronym